mlflow/server/js/src/model-registry/components/ModelVersionView.js

```diff
@@ -316,8 +316,15 @@ export class ModelVersionViewImpl extends React.Component {
         </a>
       );
     } else if (runInfo) {
+      const artifactPath = modelSource.substring(modelSource.indexOf('/artifacts/') + 11);
```

Comment:
@dbczumar Test the case where source run is defined but `source` lives outside the run (strange case, but could happen)

Reply:
@dbczumar Any updates for this comment ? :)

Reply:
In latest code, if model source is empty or parsing model source failed, it will still jump to the run page. So for some edge case the page won't be broken.